commit 28c83737345e9b9547a87105ab4d32b6f636bd4b
Author: Kefu Chai <kchai@redhat.com>
Date:   Thu Jun 8 11:21:11 2017 +0800

    msg,common: do not include assert.h everywhere to clobber <assert.h>
    
    * reverts c93daaf92b643a06a822558973d377733dbd9d85
    * add "#include "include/assert.h" to where <cassert> or <assert.h> are
      included, or avoid including them if possible.
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index e27fff4fffa..282bd4f37ec 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -25,9 +25,6 @@
 #include "messages/MCommandReply.h"
 #include "messages/MPGStats.h"
 
-// reinclude our assert to clobber the system one
-#include "include/assert.h"
-
 #define dout_subsys ceph_subsys_mgrc
 #undef dout_prefix
 #define dout_prefix *_dout << "mgrc " << __func__ << " "
